Microsoft.Toolkit.Uwp.SampleApp/Shell.SamplePicker.cs

```diff
@@ -125,7 +126,7 @@ private async void ShowSamplePicker(Sample[] samples = null, bool group = false)
 
         private void NavView_ItemInvoked(Microsoft.UI.Xaml.Controls.NavigationView sender, Microsoft.UI.Xaml.Controls.NavigationViewItemInvokedEventArgs args)
         {
-            if (args.InvokedItem is SampleCategory category)
+            if (args.InvokedItem is FrameworkElement fe && fe.DataContext is SampleCategory category)
```

Comment:
Did this never work? Or was there a breaking change that replaced the item with the element?

Reply:
They changed something in WinUI 2.4, I'm a bit concerned by this. We also have a custom style, so I'm not sure if it has something to do with that.

Reply:
@ranjeshj should we be concerned here? @anawishnoff @YuliKl

Or is this just because we have a fraken-template now?

Reply:
@ojhad might be able to give more context on this

Reply:
@michael-hawker  can you please file a bug in WinUI. You should not have to make this change.

Reply:
Yup, thanks @ranjeshj, I'll do a quick test in the morning to see if I see the same behavior in the default template, if so I'll file a bug. If not, then it must be something in our template, so I'll try from fresh and see if I still see the issue in our environment.

Reply:
@ranjeshj @ojhad @anawishnoff Yes, this indeed a regression on 2.4 causing this to break. I have an isolated repro without custom styling and detailed information in the new bug I filed [here in the WinUI repo](https://github.com/microsoft/microsoft-ui-xaml/issues/2520).

I'd expect this to break pretty much anyone using `NavigationView` from MUX upgrading to 2.4. Is this something you'd hotfix, and if so, what would the turn-around time be for that? Thanks!

Reply:
Also, if you do, I'd suggest unlisting the `2.4.0` and `2.4.2` packages on NuGet after to prevent accidental updates?

Reply:
@azchohfi I thought we had a Keyboard focus issue in the top nav, but it's another bug in the NavigationView control as well (in both 2.3 and 2.4). I filed another issue [on WinUI here for that](https://github.com/microsoft/microsoft-ui-xaml/issues/2521). FYI @ranjeshj @anawishnoff @ojhad.